src/modules/access-wallet/software/components/AccessWalletMnemonic.vue

```diff
@@ -200,9 +195,6 @@
             <div class="headline font-weight-bold">
```

Comment:
can you add "mb-5" class to Select Address and Network div (otherwise there is no space once you removed bottom p